packages/components/src/theme.test.js

```diff
@@ -0,0 +1,43 @@
+import { getTheme } from './theme';
+
+describe('Theme tool', () => {
+	const cssModule1 = {
+		prop: 'prop_mod1',
+		prop2: 'prop2_mod1',
+		'test-multiple-classes': 'test-multiple-classes-mod1',
+	};
+	const cssModule2 = {
+		prop: 'prop_mod2',
+		'test-multiple-classes': 'test-multiple-classes-mod2',
+	};
+	const cssTheme = getTheme(cssModule1, cssModule2);
+
+	it('should add the default theme classes', () => {
+		// given
+		const className = 'prop';
+		// when
+		const result = cssTheme(className);
+		// then
+		expect(result).toEqual('prop prop_mod1 prop_mod2');
+	});
+
+	it('should add the theme classes with conditions', () => {
+		// given
+		// when
+		const result = cssTheme('toto', { prop: false, prop2: true });
```

Comment:
can you add a UT with the explicit case of an Array for your second parameter as its typeof is also Object ?